uk.co.compucorp.civicrm.tasksassignments/views/dashboard/calendar.taskList.html

```diff
@@ -68,24 +68,6 @@ <h4 class="header-bar header-bar-dark">Tasks:</h4>
       </p>
     </div>
     <div class="col-xs-5">
-      <span class="{{prefix}}assignment-type">
```

Comment:
Did you check this markup was present? It looks like so much time was invested to write this code, it does not look to me that it was written by mistake. 
Please find out what kind of data this element shows and do we really need to remove this al together.